ui/app/AppLayouts/Chat/controls/community/HoldingsDropdown.qml

```diff
@@ -0,0 +1,263 @@
+                        Layout.fillWidth: true
+                        //Layout.preferredHeight: 36
+                        text: root.tokenAmountValue
+                        font.pixelSize: 13
+                        rightPadding: amountText.implicitWidth + amountText.anchors.rightMargin + leftPadding
+                        input.placeholderText: "0"
+                        validationMode: StatusInput.ValidationMode.IgnoreInvalidInput
+                        validators: StatusFloatValidator {  bottom: 0 }
+
+                        StatusBaseText {
+                            id: amountText
+                            anchors.right: parent.right
+                            anchors.rightMargin: 13
+                            anchors.verticalCenter: parent.verticalCenter
+                            text: qsTr("Amount")
+                            color: Theme.palette.baseColor1
+                            font.pixelSize: 13
+                        }
+                        onTextChanged: root.tokenAmountValue = text
+                    }
+                    // Just a filler
+                    Item { Layout.fillHeight: true}
+                    // TODO: Needed `StatusButton` redesign that allows to fill the width.
+                    StatusButton {
+                        enabled: d.ready
+                        text: qsTr("Add")
+                        height: 44
+                        Layout.alignment: Qt.AlignHCenter
+                        //Layout.fillWidth: true
+                        onClicked: { root.addToken(root.tokenAmountValue + " " + root.tokenName, root.tokenImage, root.operator) }
+                    }
+                } // End of Tokens Layout definition
+
+                // TODO
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                }
+
+                // TODO
+                Rectangle {
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                }
+            }
+        }
+    }
+
+    Component {
+        id: operatorsSelectorView
+        ColumnLayout {
+            anchors.fill: parent
+            anchors.margins: 8
+            StatusPickerButton {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 36
+                bgColor: Theme.palette.primaryColor3
+                contentColor: Theme.palette.primaryColor1
+                text: qsTr("+ And...")
+                textPixelSize: 13
+                onClicked: {
+                    root.operator = Utils.Operators.And
+                    loader.sourceComponent = tabsView
+                }
+            }
+            StatusPickerButton {
+                Layout.fillWidth: true
+                Layout.preferredHeight: 36
+                bgColor: Theme.palette.primaryColor3
+                contentColor: Theme.palette.primaryColor1
+                text: qsTr("% Or...")
```

Comment:
What does this "%" sign mean?

Reply:
Leftover! `Or-Condition` icon added and also `Add` icon one.